Interni/NormeProgetto/components/norme-c2.tex

```diff
@@ -0,0 +1,205 @@
+\section{Processi primari}
+\subsection{Fornitura}
+\subsubsection{Scopo}
+La fornitura rappresenta ciò che il fornitore svolge al fine di capire e soddisfare appieno quello che viene richiesto dal proponente.  In particolare,  il fornitore svolge un'analisi approfondita,  alla quale consegue la stesura dello \SdF{}, in modo da determinare criticità e rischi legati alla richiesta d'appalto. Inoltre, è necessario che venga definito un accordo sotto forma di contratto, così da regolarizzare la corrispondenza col proponente e tutto ciò che è legato col prodotto finale, tra cui consegna e manutenzione. A questo punto, si passa al determinare le procedure e le risorse necessarie,  sviluppandole nel \PdP{}. Quest'ultimo pone le basi per la realizzazione del prodotto, fino alla consegna finale.\newline 
+Il processo di fornitura è composto dalle seguenti fasi:
+\begin{itemize}
+	\item avvio;
+	\item approntamento di risposte alle richieste;
+	\item contrattazione;
+	\item pianificazione;
+	\item esecuzione e controllo;
+	\item revisione e valutazione;
+	\item consegna e completamento.
+\end{itemize}
+\subsubsection{Descrizione}
+Il documento include le norme che i membri del gruppo \Gruppo{} devono fare riferimento al fine di ottenere il ruolo di fornitori del proponente \proponente{} e dei committenti \textit{Prof. Tullio Vardanega} e \textit{Prof. Riccardo Cardin}. I membri del gruppo sono tenuti ad attenersi a quanto scritto nella suddetta sezione durante tutte le fasi di sviluppo del prodotto \progetto{}.
+\subsubsection{Aspettative}
+Durante lo svolgimento del progetto il gruppo intende mantenere un dialogo costante con il proponente, in modo da instaurare un rapporto di collaborazione e ricevere un riscontro proficuo sul lavoro svolto.  In particolare, si vuole:
+\begin{itemize}
+	\item determinare gli aspetti chiave che il prodotto dovrà soddisfare;
+	\item definire vincoli e requisiti riguardanti i processi;
+	\item stimare costi e tempistiche di lavoro;
+	\item accordarsi sulla qualifica del prodotto.
+\end{itemize}
+\subsubsection{Studio di fattibilità}
+Il \RdP{} ha il compito di organizzare riunioni tra i membri in modo da promuovere lo lo scambio di opinioni sui capitolati proposti. Una volta che viene trovato un capitolato di interesse comune, gli \anas{} provvedono alla stesura dello \SdF{} che, per ogni capitolato, indica:
```

Comment:
Cambiare
"... promuovere lo lo scambio ..."
in
"... promuovere lo scambio ..."

+++++

"... Una volta che viene trovato ..."
in
"... Una volta identificato ..."

Reply:
Fixed